src/lib/detectInstall.js

```diff
@@ -60,7 +66,8 @@ function detectInstall (context) {
         directory: path,
         override: true,
         moduleName: filesystem.read(`${path}/package.json`, 'json').name,
-        type: 'directory'
+        type: 'directory',
+        global: global
```

Comment:
When using `isGlobal: isGlobal`, you can just do `isGlobal` and it'll infer the `isGlobal: ` part.